### ISSUE TITLE
Keyboard navigation doesn't work

### ISSUE THREAD

**Issue by ruthra-kumar** [frappe/helpdesk#2112]
### Information about bug

Page Up / Down, Home and End doesn't work.

### Version

develop

### Installation method

None

**Comment by aerodeval** [comment]
Reverting the fix made in https://github.com/frappe/helpdesk/pull/2969/changes/2cf6fa9df987e73e9a45ead01610f447690faf6f as it is breaking Filters functioning. Need to use a different approach

**Comment by RitvikSardana** [comment]
resolved
